install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -759,23 +758,29 @@ func (config *Config) SetConsoleLink(ctx context.Context, client client.Client,
 			}
 		}
 
-		if err := reconcileConsoleLink(ctx, syndesis, syndesisRouteHost, consoleLink, client); err != nil {
+		if err := config.reconcileConsoleLink(ctx, syndesis, syndesisRouteHost, consoleLink, client); err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
-func reconcileConsoleLink(ctx context.Context, syndesis *synapi.Syndesis, routeHost string, link *consolev1.ConsoleLink, client client.Client) error {
+func (config *Config) reconcileConsoleLink(ctx context.Context, syndesis *synapi.Syndesis, routeHost string, link *consolev1.ConsoleLink, client client.Client) error {
 	updateConsoleLink := false
 	url := "https://" + routeHost
 	if link.Spec.Href != url {
 		link.Spec.Href = url
 		updateConsoleLink = true
 	}
 
-	linkText := syndesis.Name
+	var linkText string
+	if config.Productized {
+		linkText = "Fuse Online [" + syndesis.Name + "]"
```

Comment:
This should not be appearing upstream. We should just delegate to a string in a file and that file contains the product name downstream.

Reply:
Oh, yeah, I will change this one downstream.